fixieai/cli/agent/commands.py

```diff
@@ -451,18 +448,26 @@ def _validate_agent_loads_or_exit(
 )
 @click.option(
     "--venv/--no-venv",
-    "use_venv",
+    "use_venv_flag",
     is_flag=True,
     default=False,
-    help="Run from virtual environment",
+    help="Run from virtual environment [Only used for Python agents]",
```

Comment:
I think we'll continue to have awkwardness around flags that are only relevant for one language, but we can iron that out later.